controllers/alerts/secret.go

```diff
@@ -0,0 +1,104 @@
+package alerts
```

Comment:
I'm not sure I understand why we need to store the token in a secret. First, this is a security risk - not sure we need this exposure. In addition, the only usage (I found) is for e2e tests, and we don't use it for production.

So unless I miss something, I would drop this secret.